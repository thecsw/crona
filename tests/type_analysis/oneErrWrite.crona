main : void() {
    p : bool;
    write p + 1;
}
