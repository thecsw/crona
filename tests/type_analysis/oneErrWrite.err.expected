FATAL [3,11]: Arithmetic operator applied to invalid operand
Type Analysis Failed